book/01-introduction/sections/installing.asc

```diff
@@ -1,83 +1,83 @@
+Ama bunun tam olarak Git olmadığını, indirdiğiniz şeyin Windows için Git adlı bir proje olduğunu ve kendisinin Git'den farklı olduğunuzu unutmayın. Daha fazla bilgi için: https://gitforwindows.org[]
 
-To get an automated installation you can use the https://chocolatey.org/packages/git[Git Chocolatey package].
-Note that the Chocolatey package is community maintained.
+Otomatik bir kurulum için şu bağlantıyı kullanabilirsiniz: https://chocolatey.org/packages/git[Git Chocolatey package]
+Chocolatey paketinin topluluk tarafından geliştirildiğini ve denetlendiğini unutmayın.
 
-Another easy way to get Git installed is by installing GitHub Desktop.
-The installer includes a command line version of Git as well as the GUI.
-It also works well with PowerShell, and sets up solid credential caching and sane CRLF settings.(((PowerShell)))(((CRLF)))(((credential caching)))
-We'll learn more about those things a little later, but suffice it to say they're things you want.
-You can download this from the https://desktop.github.com[GitHub Desktop website].
+Git'i yüklemenin bir diğer kolay yolu da GitHub Masaüstü aracılığıyla yüklemektir.
+Bahsi geçen kurucu GKA'nün yanında Git'in komut satırı versiyonunu da içerir.
+Ayrıca PowerShell ile de epey iyi çalışır ve sets up solid credential caching and sane CRLF settings.(((PowerShell)))(((CRLF)))(((credential caching)))
+Bunlar hakkında ileride daha çok şey öğreneceğiz ama bunların ihtiyaç duyduğunuz şeyler olduğunu belirtmekte fayda var.
+Şu bağlantıdan indirebilirsiniz: https://desktop.github.com[GitHub Desktop website]
 
-==== Installing from Source
+==== Kaynaktan Yüklemek
 
-Some people may instead find it useful to install Git from source, because you'll get the most recent version.
-The binary installers tend to be a bit behind, though as Git has matured in recent years, this has made less of a difference.
+Bazı insanlar Git'i, en güncel sürümü edinebildiğiniz için, bizzat kaynağından yüklemenin daha faydalı olduğunu düşünebilir.
+İkili yükleyiciler biraz geriden gelmeye meyillidir ama Git yıllar içinde olgunlaştıkça fark git gide azaldı.
 
-If you do want to install Git from source, you need to have the following libraries that Git depends on: autotools, curl, zlib, openssl, expat, and libiconv.
-For example, if you're on a system that has `dnf` (such as Fedora) or `apt-get` (such as a Debian-based system), you can use one of these commands to install the minimal dependencies for compiling and installing the Git binaries:
+Eğer Git'i kaynaktan yüklemek istiyorsanız Git'in bel bağladığı şu kütüphanelere sahip olmalısınız: autotools, curl, zlib, openssl, expat ve libiconv.
+Örneğin eğer `dnf` (Fedora gibi) sahibi bir sistem kullanıyorsanız ya da `apt-get` (Debian-bazlı sistem gibi), derleme yapmak için gereken en temel bağımlılıkları ve Git binary'lerini yüklemek için şu aşağıdaki komutlardan birini kullanabilirsiniz:
```

Comment:
`Git binary'lerini` icin de guzel bir sey bulmaya calisalim. Belli ki `binary` kelimesini dogru yerellestirememis, sadece sayisal anlamiyla alip diger her seyi kenara itmisiz bu gune kadar.

Reply:
https://python-istihza.yazbel.com/ikili_dosyalar.html

Mesela Yazbel, binary'i direkt "ikili" olarak çevirmiş. Eğer bildiğiniz kaliteli, isabetli ve uygun bir karşılık varsa onu kullanalım.

Reply:
Doğru karşılık: git çalıştırılabiliri
Söylemesi zor gelebilir ama söyleye söyleye alışılıyor :-)
Bu arada "dosya bazlı" yerine "dosya temelli" daha türkçe bir karşılık olurdu.
dependency: bağımlılık (Git'in bağımlı olduğu şu kütüphaneler...)
'dnf' sahibi  - iyi olmamış, 'dnf' ya da  'apt-get' kullanılan bir sistemde, derleme ...
Sizli bizli çeviriler yapmamaya çalışın, öyle yazılmış olsalar bile. Çeviriler mümkünse şahıssız olsun.